public/sass/custom/kth-klaro.scss

```diff
@@ -58,96 +58,95 @@
   .cookie-notice {
     &:not(.cookie-modal-notice) {
       position: fixed;
-      right: auto;
-      top: auto;
-      left: 20px;
-      bottom: 20px;
-      background: none;
-      border-radius: 0;
-      z-index: 99999;
-
-      .cn-body {
-        font-size: 1rem;
-        padding: 1.5rem;
-        padding-top: 0.5rem;
-        position: absolute;
-        width: 554px;
-        height: fit-content;
-        max-width: 554px;
-        background-color: $white;
-        border-radius: 0.25rem;
-        box-shadow: 0 2px 10px 2px rgb(0 0 0 / 20%), 5px 5px 10px 0 rgb(0 0 0 / 19%);
-
-        .cn-ok {
-          flex-flow: column !important;
+      font-size: 1rem;
+      padding: 1.5rem;
+      margin: 10px;
+      padding-top: 0.5rem;
+      width: 554px;
+      height: fit-content;
+      max-width: 554px;
+      background-color: #fff;
+      border-radius: 0.25rem;
+      box-shadow: 0 2px 10px 2px rgba(0, 0, 0, 0.2), 5px 5px 10px 0 rgba(0, 0, 0, 0.19);
+      border: none;
+      bottom: 0;
+      right: 0;
+      left: 0;
+      #id-cookie-notice {
+        color: black;
+      }
+      .cn-ok {
+        flex-flow: column !important;
+      }
+      .cm-link {
+        &.cn-learn-more {
+          color: #007fae;
+          margin-bottom: 10px;
         }
+      }
+      .cn-buttons {
+        @media (max-width: 575px) {
+          margin-top: 0.5rem;
+        }
+        margin-top: 1rem;
 
-        .cn-buttons {
-          button {
-            &.cm-btn {
-              @media (max-width: 384px) {
-                width: auto;
-              }
+        button {
+          &.cm-btn {
+            @media (max-width: 384px) {
+              width: auto;
+            }
 
-              &.cm-btn-danger {
-                &.cn-decline {
-                  min-height: 2.75rem;
-                  color: #000;
-                  background-color: #fff;
-                  border-color: #000;
-                  border: 1px solid;
-                  font-weight: bold;
-                  padding: 0.5rem 1.5rem;
-
-                  @media (max-width: 554px) {
-                    margin-bottom: 10px;
-                  }
+            &.cm-btn-success {
+              color: #fff;
+              background-color: #007fae;
+              border-color: #007fae;
+              border: 1px solid rgba(0, 0, 0, 0);
```

Comment:
Sass-variables instead for `color`, `background-color`, `border-color`, and `border`?

Wait, first `border-color` and then a color code in `border` shorthand?